crates/ruff_linter/src/rules/flake8_simplify/rules/ast_expr.rs

```diff
@@ -110,6 +110,10 @@ impl AlwaysFixableViolation for DictGetWithNoneDefault {
     }
 }
 
+fn is_lowercase_allowed(env_var: &str) -> bool {
+    matches!(env_var, "https_proxy" | "http_proxy" | "no_proxy")
```

Comment:
Could you add a comment linking to the sources/discussion for why these are lowercase?

Reply:
Added!